README.md

```diff
@@ -1,4 +1,5 @@
-# IPFS unixFS Engine
+IPFS unixFS Engine
```

Comment:
I've been (since forever) doing `====` for the title, and now it is just autopilot.